code/src/renderer/style.global.css

```diff
@@ -616,3 +616,11 @@ html {
 #doctest-timeout {
     width: 80px;
 }
+
+.scheme-editor .ace_marker-layer .ace_bracket {
+    display: none;
+}
+
+.scheme-editor .ace_marker-layer .match_parens {
+    background: #004ea2 !important; /* I hate CSS */
```

Comment:
Are you sure you need !important? You just need to be more specific than the most specific ACE rule. It seemed to work for the cases I tried without important. You could also double up the classes on the last part, if ace_selection will always be a co-class:
.scheme-editor .ace_marker-layer .ace_selection.match_parens

Reply:
You're right, `!important` isn't needed. There's an ACE rule with equal precedence, but our stylesheet comes after it in the HTML, so I guess we override it. Still, I'll give it strictly greater priority, since I don't know if the order of CSS styles is deterministic in webpack.